html/includes/table/alert-schedule.inc.php

```diff
@@ -31,7 +32,7 @@
     $total = 0;
 }
 
-if (!isset($sort) || empty($sort)) {
+if (!isset($sort) || empty($sort) || substr($sort, 1, 6) == 'status') {
```

Comment:
The ports.inc.php file in this dir does the sort by:

`list($sort_column, $sort_order) = explode(' ', trim($sort));`

It would be better to stick to the same method here if you can.

Reply:
Updated to use the same syntax that the ports.inc.php file